reader/processor/processor.go

```diff
@@ -112,8 +112,18 @@ func isAllowedEntry(feed *model.Feed, entry *model.Entry) bool {
 // ProcessEntryWebPage downloads the entry web page and apply rewrite rules.
 func ProcessEntryWebPage(feed *model.Feed, entry *model.Entry) error {
 	startTime := time.Now()
+	var url = entry.URL
+	if feed.UrlRewriteRules != "" {
+		parts := strings.Split(feed.UrlRewriteRules, "@@@")
```

Comment:
Any particular reason why you choose `@@@` as separator?

Reply:
Not really, this is just a pattern I use often when I need a unique-enough identifier or search term. This still feels cumbersome to me, I'd like it more if I could somehow split the search and replace part of the configuration into its own fields...

Reply:
I will have a look at `rewrite.Rewriter` and try to mimic the approach.

Reply:
I rewrote the URL rewrite to use a regex similar to `rewrite.Rewriter`. This looks better to me and it also matches already existing functionality.